### ISSUE TITLE
Not liking more than 27 photos

### ISSUE THREAD

**Issue by Fellowes52** [InstaPy/InstaPy#1403]
Instapy was working perfectly on my Raspberry Pi 3 for ages. Now all of a sudden it's only liking a max of 27 photos per hashtag that I set.

I've even tried this on Windows 10 running the same files and again it limits each hashtag to 27 max likes!

Any ideas on how to fix this? Before it would like what ever I set as the peripheral.

**Comment by ghost** [comment]
Ive got the same issue now, except with 12 images..
Looking forward to a solution.

**Comment by riccardorighettiphoto** [comment]
Hello,
just in case you wanna try, I followed this [fix](https://github.com/timgrossmann/InstaPy/issues/1365#issuecomment-363586896). Please, pay strong attention at code indentation because I did many mistakes by copying the code via Notepad++ and only a bunch of minutes ago I could apply the fix correctly.
Now I am running the script on 100 pictures.

Bye.

**Comment by FranklinClinton** [comment]
@riccardorighettiphoto Does not work for me. Still only liking 27 Pictures. Log:
```
2018-02-11 13:35:31 - INFO - Session started - 2018-02-11 13:35:31
2018-02-11 13:35:34 - INFO - Logged in successfully!
2018-02-11 13:35:35 - INFO - Tag [1/1]
2018-02-11 13:35:35 - INFO - --> b'got'


2018-02-11 13:39:36 - INFO - This tag has less pictures than intended..
2018-02-11 13:39:41 - INFO - [1/27]
```

**Comment by Fellowes52** [comment]
Hi @riccardorighettiphoto 

Thanks for the link to fix however the script is now completely broken! I tried to do a new clone as "like_util.py" was last updated 2 days ago. Hopefully it's fully fixed in the next few days so as people can use it all again.

**Comment by chrisscholly** [comment]
Same issue here.
Looking forward to a solution too.

**Comment by uluQulu** [comment]
@chrisscholly have you tried [this](https://github.com/timgrossmann/InstaPy/issues/1365#issuecomment-363586896)? (i've just updated)

**Comment by chrisscholly** [comment]
Currently trying, thank you @uluQulu :)

**Comment by chrisscholly** [comment]
@uluQulu Seems like it works like a charm 🎉
Thank you very much :)

**Comment by imewish** [comment]
Cant, we get an official fix for this with the above fix from @uluQulu?

**Comment by dimahadgi** [comment]
Got the same issue. Only 27 likes.

**Comment by uluQulu** [comment]
hi Dimitriy, get the solution from the link I shared above, it works for many!

**Comment by dimahadgi** [comment]
Hi @uluQulu . Thanks, your fix working perfectly. Could you please do the same for: 
`def get_links_for_location` ? 
Only for that reason I am using this bot.

**Comment by uluQulu** [comment]
@dimahadgi I guess they are so similar, will try to look at it soon

**Comment by dimahadgi** [comment]
Tried to figure out how it works and just copy your fixed code to def get_links_for_location.
Looks like it works perfect, but need to test.
Here you can get full like_util.py
https://pastebin.com/AsJXpK97

**Comment by timgrossmann** [comment]
Should be fixed by #1844
